global-script.lua

```diff
@@ -3079,8 +3087,30 @@ function tCompare(t1,t2)
         newTab = {}
         for i,v in ipairs(tab) do
             newTab[i] = table.concat(v,",")
-    end
-    return table.concat(newTab,"|")
+        end
+        return table.concat(newTab,"|")
     end
     if cc2(t1) == cc2(t2) then return true else return false end
-end
\ No newline at end of file
+end
+function togglePlayer(player, color)
+    player = Player[player.color]
+    player.changeColor(color)
+    Wait.frames(function ()
+        Player[color].lookAt({
+            position = getObjectFromGUID(playerCounters[color]).getPosition() - Vector(0,0,15),
+            pitch = 70,
+            distance = 30,
+        })
+    end, 2)
+end
+function onPlayerChangeColor(color)
```

Comment:
you probably want to extract this code out into a separate function and then call that here for `onPlayerChangeColor` as well as `onLoad` and when the game starts